sarama.go

```diff
@@ -22,6 +24,10 @@ import (
 // but you can set it to redirect wherever you want.
 var Logger StdLogger = log.New(ioutil.Discard, "[Sarama] ", log.LstdFlags)
 
+// Random is a random number generator instance used throughout sarama for random behavior.
```

Comment:
we're not using it for random partitioner instances, should we? it then becomes a single point of contentiont